### ISSUE TITLE
Missing variable in forward_sync

### ISSUE THREAD

**Issue by yzmyyff** [mindslab-ai/phaseaug#10]
I got an exception from forward_sync:
```
...
  File "/usr/local/lib/python3.8/dist-packages/phaseaug/phaseaug.py", line 117, in forward_sync
    phi = self.sample_phi(self, X.shape[0] // 2) #[2B, nfft//2+1]
NameError: name 'X' is not defined
```

X is the result of stft and is removed in version 1.0.0

**Comment by junjun3518** [comment]
Thank you for pointing out my mistakes!
I fixed it and released as version 1.0.1

**Comment by yzmyyff** [comment]
Thanks a lot.
Version 1.0.1 worked.
